modules/pytket-quantinuum/examples/Quantinuum Circuit Submissions via pytket.ipynb

```diff
@@ -1237,11 +1241,7 @@
     }
    ],
    "source": [
-    "machine = 'H1-1E'\n",
-    "\n",
-    "backend = QuantinuumBackend(device_name=machine)\n",
-    "\n",
-    "print(machine, \"status:\", backend.device_state(device_name=machine))"
+    "print(machine, \"status:\", QuantinuumBackend.device_state(device_name=machine))"
```

Comment:
Have you check that this is changed correctly? Why are you not using the backend you created?

Reply:
Okay, would have been nice to do the status check fist before selecting a device, but I would assume that is not possible with the current API.

Reply:
Yes, the current implementation of login has some issues. That will be addressed in a future PR.